src/Network/Probe.php

```diff
@@ -333,7 +333,7 @@ public static function uri($uri, $network = "", $uid = 0, $cache = true) {
 			/// The biggest problem is the avatar picture that could have a reduced image size.
 			/// It should only be updated if the existing picture isn't existing anymore.
 			if (($data['network'] != NETWORK_FEED)
-				AND ($mode == PROBE_NORMAL)
+				AND ($data["network"] != NETWORK_DIASPORA)
```

Comment:
BTW: You have to answer if this is/was the correct way of solving the issue. I'm not familiar with this part of the code: I have no idea what behavior was intended and why.